pkgs/development/tools/misc/cmake-language-server/default.nix

```diff
@@ -11,17 +11,17 @@
 
 buildPythonApplication rec {
   pname = "cmake-language-server";
-  version = "unstable-2023-01-08";
+  version = "0.1.7";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "regen100";
-    repo = pname;
-    rev = "60c376a5fda29835060687569cb212350a292116";
-    hash = "sha256-vNG43sZy2wMetY5mbgxIoei5jCCj1f8vWiovWtwzbPc=";
+    repo = "cmake-language-server";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
I stopped using `pname` for fetchers in the packages I my maintain because this coupling doesn't always make sense, the source repo should always be the same even if the pname changes.